app/models/form_submission_attempt.rb

```diff
@@ -33,6 +33,8 @@ class FormSubmissionAttempt < ApplicationRecord
                               form_type: form_submission.form_type
                             })
           simple_forms_enqueue_result_email(:error)
+          tags = ['service:veteran-facing-forms', "function:#{form_id} form submission to Lighthouse"]
```

Comment:
Is `form_id` what you intend to interpolate here? If so, where is the variable defined?

Reply:
Ah! Good catch. I fixed it. Thanks!